### PR TITLE
Add Black Angus Steakhouse restaurant

### PR DESCRIPTION
**Pull request by felipehfs**

### CODE REVIEW

File: brands/amenity/restaurant.json [hunk 1]
```diff
@@ -174,6 +174,17 @@
       "name:he": "בלאק"
     }
   },
+  "amenity/restaurant|Black Angus": {
+    "countryCodes": ["en"],
```
Comment (bhousel) [inline brands/amenity/restaurant.json]:
```suggestion
    "countryCodes": ["us"],
```

File: brands/amenity/restaurant.json [hunk 2]
```diff
@@ -174,6 +174,17 @@
...
+    "tags": {
+      "amenity": "restaurant",
+      "brand": "Black Angus",
+      "brand:wikidata": "Q4920269",
+      "brand:wikipedia": "en:Black_Angus_Steakhouse",
```
Comment (bhousel) [inline brands/amenity/restaurant.json]: these links need underscores replaced with spaces..
```suggestion
      "brand:wikipedia": "en:Black Angus Steakhouse",
```

File: brands/amenity/restaurant.json (hunk 1 above)
Comment (felipehfs) [reply brands/amenity/restaurant.json]: Ok

File: brands/amenity/restaurant.json (hunk 2 above)
Comment (felipehfs) [reply brands/amenity/restaurant.json]: ok

**Comment by bhousel** [comment]
Great @felipehfs , thanks for the quick fixes and welcome! 👋

**Pull request merged**